src/main/java/mainclasses/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package mainclasses;
```

Comment:
Should the pakage name be more specific? For example, those about UI can be undert the package ui and those about storage can be about storage.